pacbio-subreads-processing.wdl

```diff
@@ -218,3 +267,69 @@ workflow SubreadsProcessing {
         refineStderr: {description: "Refine stderr log file(s)."}
     }
 }
+
+task ccsChunks {
+    input {
+        Int chunkCount
+        String dockerImage = "python:3.7-slim"
+        String memory = "4G"
+    }
+
+    command {
+        set -e
+        python <<CODE
+
+        for i in range(1, ~{chunkCount} + 1):
+            print(i, ~{chunkCount}, sep="/")
+        CODE
+    }
+
+    runtime {
+        docker: dockerImage
+        memory: memory
+    }
+
+    output {
+        Array[String] chunks = read_lines(stdout())
+    }
+
+    parameter_meta {
+        chunkCount: {description: "The number of chunks to create.", category: "required"}
+        dockerImage: {description: "The docker image used for this task.  Changing this may result in errors which the developers may choose not to address."}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+    }
+}
+
+task mergePacBio {
+    input {
+        Array[File]+ reports
+        String mergedReport
+        String dockerImage = "lumc/pacbio-merge:0.2"
+        String memory = "4G"
+    }
+
+    command {
+        set -e
+        mkdir -p $(dirname ~{mergedReport})
+        pacbio_merge \
+            --reports ~{sep=" " reports} \
+            --json-output ~{mergedReport}
+    }
+
+    runtime {
+        docker: dockerImage
+        memory: memory
+    }
+
+    output {
+        File MergedReport = mergedReport
+    }
+
+    parameter_meta {
+        # inputs
+        reports: {description: "The PacBio report files to merge.", category: "required"}
+        mergedReport: {description: "The location the merged PacBio report file should be written to.", category: "common"}
+        memory: {description: "The amount of memory this job will use.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.", category: "advanced"}
+    }
+}
```

Comment:
Are these now in the `tasks` repo as well?

Reply:
No, they are too obscure to put in the tasks repo, I don't think anyone else is ever going to use these.